shared/src/main/scala/com/all4journey/shared/domain/User.scala

```diff
@@ -1,8 +1,10 @@
 package com.all4journey.shared.domain
 
 import com.github.t3hnar.bcrypt._
+import com.wix.accord._
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Use brackets to group imports from the same package](https://www.codacy.com/app/rjkernick/all4journey/file/2652623797/issues/source?bid=3246460&fileBranchId=3325235#l4)